docs/source-pytorch/advanced/third_party/horovod.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
+
+#######
+Horovod
+#######
+
+The `Horovod strategy <https://github.com/Lightning-AI/lightning-horovod>`_ allows the same training script to be used for single-GPU, multi-GPU, and multi-node training.
+
+.. warning::  This is an :ref:`experimental <versioning:Experimental API>` feature.
+
+Like Distributed Data Parallel, every process in Horovod operates on a single GPU with a fixed subset of the data.  Gradients are averaged across all GPUs in parallel during the backward pass, then synchronously applied before beginning the next step.
+
+The number of worker processes is configured by a driver application (`horovodrun` or `mpirun`). In the training script, Horovod will detect the number of workers from the environment, and automatically scale the learning rate to compensate for the increased total batch size.
+
+You can install the Horovod integration by running
+
+.. code-block:: bash
+
+    pip install lightning-horovod
+
+This will install both the `Horovod <https://github.com/horovod/horovod#install>`_ package as well as the ``HorovodStrategy`` for the Lightning Trainer:
+
+.. code-block:: python
+
+    from lightning import Trainer
+    from lightning_horovod import HorovodStrategy
+
+    # train Horovod on GPU (number of GPUs / machines provided on command-line)
+    trainer = Trainer(strategy="horovod", accelerator="gpu", devices=1)
+
+
```

Comment:
```suggestion
This will install both the `Horovod <https://github.com/horovod/horovod#install>`_ package as well as the ``HorovodStrategy`` for the Lightning Trainer.
```